HLTrigger/Muon/src/HLTL1MuonNoL2Selector.cc

```diff
@@ -53,7 +53,6 @@ HLTL1MuonNoL2Selector::fillDescriptions(edm::ConfigurationDescriptions& descript
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("InputObjects",edm::InputTag(""));
   desc.add<edm::InputTag>("L2CandTag",edm::InputTag("hltL2MuonCandidates"));
-  desc.add<edm::InputTag>("L1CandTag",edm::InputTag(""));
```

Comment:
What do you mean? The jenkins PR tests pass, so the module is not yet used.
So the PR can get integrated, and we can parse it into ConfDB later.